README.md

```diff
@@ -117,7 +121,8 @@ $ rails new . -d postgresql
 ```
 
 ```
-Your answer...
+The first command creates the rails files, but with the default sqlite database.
+The second creates rails files along with postgresql database
 ```
 
```

Comment:
:+1:  the difference is the `.` means create the app in the currently directory using the current directory as the name for the app.